### PR TITLE
Test cmdline policy delete tagging

### PR DESCRIPTION
**Pull request by lrm25**

### CODE REVIEW

**Review by anodelman — CHANGES_REQUESTED** [review]
Please do a pass to remove the if-exit-code-then-return-1 with asserts.

File: tests/commands/get_bucket_tagging.sh [hunk 1]
```diff
@@ -29,4 +25,28 @@ get_bucket_tagging() {
...
+      export tags=
+      return 0
+    fi
+    fail "unrecognized error getting bucket tagging with user: $tags"
+    return 1
+  fi
+  log 5 "raw tags data: $tags"
+  tags=$(echo "$tags" | grep -v "InsecureRequestWarning")
+  log 5 "modified tags data: $tags"
+  export tags
```
Comment (anodelman) [inline tests/commands/get_bucket_tagging.sh]: Unnecessary export.

File: tests/commands/get_object_lock_configuration.sh [hunk 2]
```diff
@@ -6,11 +6,13 @@ get_object_lock_configuration() {
     log 2 "'get object lock configuration' command missing bucket name"
     return 1
   fi
-  lock_config=$(aws --no-verify-ssl s3api get-object-lock-configuration --bucket "$1") || local get_result=$?
-  if [[ $get_result -ne 0 ]]; then
+  if ! lock_config=$(aws --no-verify-ssl s3api get-object-lock-configuration --bucket "$1" 2>&1); then
     log 2 "error obtaining lock config: $lock_config"
+    get_object_lock_config_err=$lock_config
+    export get_object_lock_config_err
```
Comment (anodelman) [inline tests/commands/get_object_lock_configuration.sh]: Unnecessary export.

**Review by anodelman — CHANGES_REQUESTED** [review]
Bunch of exports to pull out.

**Comment by lrm25** [comment]
Changes that were supposed to be in this one are now in test_cmdline_policy_delete_tagging_two

**Pull request closed without merging**